src/lucky_record/query_builder.cr

```diff
@@ -86,6 +86,9 @@ class LuckyRecord::QueryBuilder
   end
 
   def count
+    if @limit || @offset
+      raise "Counting with limit or offset is not supported yet. Try calling `results.size` to count in Crystal instead."
```

Comment:
After reading this a bit more, I think `results.size` might not be that clear. What do you think about `Try calling 'results.size' on your query to count using Crystal instead of the database.` Thoughts?

Reply:
I'm all for more clarity :+1: